examples/werkstatt/werkstatt_envfollower.js

```
@@ -0,0 +1,91 @@
+
+    paramChanged(label, value) {
+        if (label === "attack") this.attack = value
+        else if (label === "release") this.release = value
+        else if (label === "depth") this.depth = value
+        else if (label === "threshold") this.threshold = value
+        else if (label === "invert") this.invert = value
+        else if (label === "makeup") this.makeup = value
+    }
+
+    process(io, block) {
+        const sr = globalThis.sampleRate || 44100
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)